comms/dht/src/network_discovery/on_connect.rs

```diff
@@ -91,7 +91,7 @@ impl OnConnect {
                                 .connectivity
                                 .ban_peer_until(
                                     conn.peer_node_id().clone(),
-                                    Duration::from_secs(60 * 60),
+                                    Duration::from_secs(DURATION_SYNC_PEERS),
```

Comment:
Dont make it a const, take it from the config, there is a config settings, `ban_short_duration`

Reply:
Great! I add this config value. Much better now :)